Sources/_TestingInternals/Discovery.cpp

```diff
@@ -213,24 +213,8 @@ struct SWTTypeMetadataRecord {
   }
 };
 
-#if defined(SWT_NO_DYNAMIC_LINKING)
```

Comment:
Moved this bit down to clarify it's "Apple with static linking" rather than _all_ static-linked cases. The code isn't changed.